src/GUI_draft/templates/create_project.html

```diff
@@ -5,6 +5,15 @@
 <!-- WRITE HTML CODE WITHIN THESE block content TAGS -->
 {% block content %}
 
+{% set project_parameters={
```

Comment:
I think the structure should be a list of dicts (you can move the name inside with the key `name` or `label` for example